news-bundle/src/Feed/FeedProviderInterface.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\NewsBundle\Feed;
+
+use Contao\Model\Collection;
+use Contao\PageModel;
+use FeedIo\FeedInterface;
+use Symfony\Component\HttpFoundation\Request;
+
+interface FeedProviderInterface
```

Comment:
maybe I'm missing something, but can you explain what the `FeedProvider(Interface)` is for since we also have events?

Reply:
You are right, this code is obsolete, since we only use events now.